autoscheduler/frontend/templates/index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <title> Auto Aggie Scheduler </title>
     </head>
-    <body>
+    <body style="background-color:#c2c2c2;">
```

Comment:
Instead of changing the background color here, change it within React.

Reply:
That would be ideal, but we will need to make at least one modification to `index.html`, and that is to give the root div control of the entire screen. At present, body gives itself an 8px margin, and because of that, there is an 8px strip around the edge of the screen that React cannot control. If we just add one style rule to body saying `margin: 0`, then all other styling can be done in React, including the background color